app/views/assignments/show.html.erb

```diff
@@ -30,10 +30,12 @@
         <%= @assignment.graded %>
       </p>
 
+     <% if @assignment.restricted_circuit_elements != "None" %>
       <p>
         <strong>Description:</strong>
         <p><%= @assignment.description&.html_safe%></p><br>
       </p>
+    <% end %>
 
       <p>
         <strong>Restricted Circuit Elements:</strong>
```

Comment:
Shouldnt this be within the if condition?

Reply:
It seems you have added the condition to the description...

Reply:
I haven't seen that too, good point